task/upload.go

```diff
@@ -583,6 +600,17 @@ func outputLocations(outURL, hls, hlsRelPath, mp4, mp4RelPath string, sourceCopy
 				},
 			},
 		}
+	if fmp4 == OUTPUT_ENABLED {
+		names, locations =
+			append(names, OutputNameEmpty),
+			append(locations, clients.OutputLocation{
+				Type: "object_store",
+				URL:  url.JoinPath(videoFileName(fmp4RelPath)).String(),
```

Comment:
@mjh1 this is adding a /video/fmp4 to the paths which we don't want. We just want the /fmp4 appended to whatever path the user provides.